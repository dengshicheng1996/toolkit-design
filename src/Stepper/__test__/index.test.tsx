import * as React from 'react'
import { shallow } from 'enzyme'
import toJSON from 'enzyme-to-json'
import { Stepper } from '..'

describe('Stepper', () => {
  it('snapshot', () => {
    const component = shallow(
      <Stepper max={10} min={1} value={1} data-y="111" />,
    )

    expect(toJSON(component)).toMatchSnapshot()
  })
})
