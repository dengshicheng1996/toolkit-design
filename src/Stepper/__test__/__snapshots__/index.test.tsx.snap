// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Stepper snapshot 1`] = `
<ForwardRef
  data-y="111"
  defaultValue={0}
  downHandler={
    <MinusOutline
      size="md"
    />
  }
  max={10}
  min={1}
  parser={[Function]}
  prefixCls="amd-stepper"
  step={1}
  upHandler={
    <AddOutline
      size="md"
    />
  }
  value={1}
/>
`;
